code/game/gamemodes/dynamic/dynamic.dm

```diff
@@ -283,13 +283,25 @@ GLOBAL_VAR_INIT(dynamic_forced_threat_level, -1)
 			. += "Your station is somehow in the middle of hostile territory, in clear view of any enemy of the corporation. Your likelihood to survive is low, and station destruction is expected and almost inevitable. Secure any sensitive material and neutralize any enemy you will come across. It is important that you at least try to maintain the station.<BR>"
 			. += "Good luck."
 
+	var/min_threat = 100
+	for(var/datum/dynamic_ruleset/ruleset as anything in subtypesof(/datum/dynamic_ruleset))
+		var/weight = initial(ruleset.weight)
```

Comment:
Don't use `initial`, otherwise these can't be configured.

Every ruleset should be stored in the dynamic datum itself, if I remember right.